### ISSUE TITLE
How can we open cash drawer alomg with the print command? 

### ISSUE THREAD

**Issue by Malikx-Alee** [HeligPfleigh/react-native-thermal-receipt-printer#24]

**Comment by arirah-int** [comment]
Did you get any solution ? @Malikx-Alee

**Comment by MaheshKaemSolutions** [comment]
Any solution for this Question @arirah-int or @Malikx-Alee

**Comment by Rogax** [comment]
NetPrinter.printBill('\x1B\x70\x00\x19\xFA');
I tried it and cut it

**Comment by fiyimira** [comment]
@Rogax do you mean calling the function: `NetPrinter.printBill('\x1B\x70\x00\x19\xFA');` opens the cash drawer?

**Comment by Rogax** [comment]
> @Rogax do you mean calling the function: `NetPrinter.printBill('\x1B\x70\x00\x19\xFA');` opens the cash drawer?

yes, i was used it to open the cash drawer. But at the moment, i am using the different package, it is react-native-thermal-receipt-printer-image-qr, it has been forked of this package.
